en/community/podcasts/index.md

```diff
@@ -18,13 +18,18 @@ Listen to news, interviews, and discussions about Ruby and its community.
 : Virtual meetup turned podcast, Remote Ruby celebrates and highlights
   the Ruby community in an informal setting.
 
+[Rooftop Ruby][rooftop_ruby]
+: Collin and Joel discuss Ruby, software development, open source, career,
+  and a lot more together and with guests.
+
 ### Getting Involved
 
 Podcast hosts are always looking for guests. If you have some Ruby
 wisdom to share, get in touch with the creators of these shows.
 
 You can also start your own Ruby podcast and get added to this list!
 
+[rooftop_ruby]: https://rooftopruby.com
```

Comment:
```suggestion
[rooftop_ruby]: https://www.rooftopruby.com
```

@collindonnell we should configure a redirect from root to `www.` but for now, it looks like the website only works on `www.`.